src/Containers/Microsoft.NET.Build.Containers/Tasks/CreateNewImage.cs

```diff
@@ -31,7 +31,19 @@ public sealed partial class CreateNewImage : Microsoft.Build.Utilities.Task, ICa
 
     public override bool Execute()
     {
-        return Task.Run(() => ExecuteAsync(_cancellationTokenSource.Token)).GetAwaiter().GetResult();
+        try
+        {
+            Task.Run(() => ExecuteAsync(_cancellationTokenSource.Token)).ContinueWith(t => t).GetAwaiter().GetResult();
```

Comment:
Please remove the task continuation if it's not necessary

Reply:
good catch, removed it now.